packages/core/src/middleware/interfaces/multer.interface.ts

```diff
@@ -0,0 +1,62 @@
+import express, { Express } from "express";  // Import Express framework
+import multer, { Multer } from "multer";
```

Comment:
You don't need multer dependency, just the types, interfaces. That's the whole point of dynamic middleware. The user decides to use the middleware, if it is installed is ready to go, otherwise it will throw a warning saying to the user install it.